source/analytics/index.html.md.erb

```diff
@@ -0,0 +1,209 @@
+---
+title: Analytics
+weight: 8
+---
+
+# Analytics
+
+We collect user data from the GOV.UK Design System website to understand how users are interacting with the design system. This helps us to make product and design decisions.
+
+## What we use for analytics
+
+### Google Analytics
+
+We use Google Analytics (GA) to analyse collected user data. We can use GA to assess a wide variety of data, such as:
+- number of visits to the Design System website over a period of time
+- most viewed pages
+
+GA also lets us visualise the data and create custom reports.
+
+We maintain 2 GA properties:
+
+- [Data from the live website](https://analytics.google.com/analytics/web/#/p446981526/reports/intelligenthome)
+- [Data from website previews, such as temporary Netlify links generated from pull requests](https://analytics.google.com/analytics/web/#/p446943224/reports/intelligenthome)
+
+### Google Tag Manager
+
+Google Tag Manager (GTM) lets us collect and configure user data and send it to our GA properties. Through our [GTM container](https://tagmanager.google.com/#/container/accounts/341497290/containers/8094879/workspaces/1000045), we can define events on the website, including:
+
+- what triggers the event
+- what information to collect
+- how to present that information
+
+These events are recorded in GA, where they can be analysed.
+
+GTM uses 3 main concepts to manage event data:
```

Comment:
I feel like I have to do some extra parsing to understand what the 3 concepts are.
```suggestion
GTM uses 3 main concepts to manage event data: tags, triggers, and variables.
```